records_mover/records/targets/table/target.py

```diff
@@ -81,22 +81,35 @@ def move_from_fileobjs_source(self,
 
     def can_move_from_fileobjs_source(self) -> bool:
         driver = self.db_driver(self.db_engine)
-        return driver.can_load_from_fileobjs()
+        loader = driver.loader_from_fileobj()
+        if loader is None:
+            return False
+        # TODO: why do we need can_load_from_fileobjs if we can just return None?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker